compact/tracking/mpgd_outerbarrel.xml

```diff
@@ -13,20 +13,21 @@
 
 <define>
     <comment> Frames </comment>
-    <constant name="MPGDOuterBarrelFrame_width"            value="10*mm"/>
+    <constant name="MPGDOuterBarrelFrame_width"            value="15*mm"/>
     <constant name="MPGDOuterBarrelFrame_thickness"        value="7*mm"/>
 
     <comment> Module constants </comment>
     <constant name="MPGDOuterBarrelModule_roverlap"                value="0*cm"/>
     <constant name="MPGDOuterBarrelModule_zoverlap"                value="0*MPGDOuterBarrelFrame_width"/>
     <constant name="MPGDOuterBarrelModule_count"                   value="12" />
-    <constant name="MPGDOuterBarrelModule_allowed_space"           value="2.5*cm"/>
+    <constant name="MPGDOuterBarrelModule_allowed_space"           value="25*mm"/>
     <constant name="MPGDOuterBarrelModule_rmax"                    value="MPGDOuterBarrelModule_rmin + MPGDOuterBarrelModule_allowed_space" />
     <constant name="MPGDOuterBarrelModule_roffset"                 value="0.5*cm" />
-    <constant name="MPGDOuterBarrelModule_width"                   value="36.0*cm"/>
+    <constant name="MPGDOuterBarrelModule_width"                   value="360*mm"/>
     <constant name="MPGDOuterBarrel_length"                        value="MPGDOuterBarrelModule_zmin1 + MPGDOuterBarrelModule_zmin2"/>
     <constant name="MPGDOuterBarrelModule_length"                  value="0.5*(MPGDOuterBarrelModule_zmin1 + MPGDOuterBarrelModule_zmin2 + MPGDOuterBarrelModule_zoverlap)"/>
-    <constant name="MPGDOuterBarrelModule_offset"                  value="(MPGDOuterBarrelModule_zmin2 - MPGDOuterBarrelModule_zmin1)/2.0"/>
+    <constant name="MPGDOuterBarrelModule_offset"                  value="(MPGDOuterBarrelModule_zmin2 - MPGDOuterBarrelModule_zmin1)"/>
+    <constant name="MPGDOuterBarrelModule_feb_ext"                 value="110*mm"/>
```

Comment:
This is used as an offset for the PCB below. The name is unclear, and it may be better to keep calling this `offset`, e.g.

```suggestion
    <constant name="MPGDOuterBarrelModule_PCB_offset"                 value="110*mm"/>
```